src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -278,6 +278,24 @@ public function getCondition()
                     'label' => $this->translator->trans('Refurbished', [], 'Shop.Theme.Catalog'),
                     'schema_url' => 'https://schema.org/RefurbishedCondition',
                 ];
+            case 'damaged':
+                return [
+                    'type' => 'damaged',
+                    'label' => $this->translator->trans('Damaged', [], 'Shop.Theme.Catalog'),
+                    'schema_url' => 'https://schema.org/DamagedCondition',
+                ];
+            case 'open_package':
+                return [
+                    'type' => 'open_package',
+                    'label' => $this->translator->trans('Opened package', [], 'Shop.Theme.Catalog'),
```

Comment:
You mentionned in the description that the product is still new. We could specify that in the label. For example, Amazona and Ebay use "New - Open box" which I think is very clear.